pr_agent/algo/utils.py

```diff
@@ -375,7 +375,7 @@ def get_user_labels(current_labels: List[str] = None):
 
 def get_max_tokens(model):
```

Comment:
**Suggestion:** Consider adding a default value for the `model` parameter in the `get_max_tokens` function to prevent potential errors if no model is provided. [possible issue]
```suggestion
def get_max_tokens(model=None):
    settings = get_settings()
    max_tokens_model = MAX_TOKENS.get(model, 4000) if model else 4000
```